test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -872,6 +894,60 @@ public void getActivityHistoryV3DefaultsToNulls() throws Exception {
         assertNull(endTimeCaptor.getValue());
     }
     
+    
+    @Test
+    public void updateIdentifiersWithPhone() throws Exception {
+        mockPlayContextWithJson(PHONE_UPDATE);
+        doReturn(session).when(controller).getAuthenticatedAndConsentedSession();
+        
+        when(mockParticipantService.updateIdentifiers(eq(study), any(), any())).thenReturn(participant);
+        
+        Result result = controller.updateIdentifiers();
+        assertResult(result, 200);
+        
+        JsonNode node = BridgeObjectMapper.get().readTree(Helpers.contentAsString(result));
+        assertEquals(ID, node.get("id").textValue());
+        
+        verify(mockParticipantService).updateIdentifiers(eq(study), contextCaptor.capture(), identifierUpdateCaptor.capture());
+        
+        IdentifierUpdate update = identifierUpdateCaptor.getValue();
+        assertEquals(EMAIL_PASSWORD_SIGN_IN_REQUEST.getEmail(), update.getSignIn().getEmail());
+        assertEquals(EMAIL_PASSWORD_SIGN_IN_REQUEST.getPassword(), update.getSignIn().getPassword());
+        assertEquals(TestConstants.PHONE, update.getPhoneUpdate());
+        assertNull(update.getEmailUpdate());
```

Comment:
Is it worth also checking that we call sessionUpdateService.updateParticipant()?